src/views/SetupKeys.tsx

```diff
@@ -354,118 +377,230 @@ export const SetupKeys = () => {
+                                    type="error"
+                                    showIcon
+                                    closable
+                                />
+                            )}
+                            <Card bodyStyle={{ padding: 0 }}>
+                                <ConfigProvider
+                                    renderEmpty={() => {
+                                        return (
+                                            <Container
+                                                style={{
+                                                    textAlign: "center",
+                                                    width: "615px",
+                                                }}
+                                            >
+                                                <Col
+                                                    style={{
+                                                        marginTop: "41px",
+                                                    }}
+                                                >
+                                                    <Text
+                                                        style={{
+                                                            fontWeight: "500",
+                                                            fontSize: "22px",
+                                                            lineHeight: "26px",
+                                                            color: "#252526",
+                                                        }}
+                                                    >
+                                                        Create Setup Key
+                                                    </Text>
+                                                </Col>
+                                                <Col
+                                                    style={{
+                                                        marginTop: "17px",
+                                                    }}
+                                                >
+                                                    <Text
+                                                        style={{
+                                                            fontWeight: "400",
+                                                            fontSize: "16px",
+                                                            lineHeight: "22px",
+                                                        }}
+                                                    >
+                                                        Manage Setup Keys to register new machines in your network. The
+                                                        key links the machine to an account during initial setup.
+                                                        <a> Learn more</a>
```

Comment:
Missing href

Reply:
could you please provide me link on what i should redirect user, since on that moment i didnt have it to paste